plugins/redshift/dbt/include/redshift/macros/adapters.sql

```diff
@@ -80,7 +80,7 @@
           numeric_precision,
           numeric_scale
 
-        from {{ relation.information_schema('columns') }}
```

Comment:
how confident are you that this is equivalent

Reply:
yeah - should be the same thing. This is definitely _not_ equivalent on BQ/Snowflake, but on Redshift, you can only access the `information_schema` for the database that you're connected to. Turns out Redshift ignores capitalization in event quoted identifiers, so all of these are equivalent 🙃 

```
select * from information_schema.columns;
select * from information_schema."columns";
select * from information_schema."COLUMNS";
```